app/bundles/LeadBundle/EventListener/LeadSubscriber.php

```diff
@@ -29,52 +29,30 @@ class LeadSubscriber implements EventSubscriberInterface
 {
     use ChannelTrait;
 
-    private \Mautic\CoreBundle\Model\AuditLogModel $auditLogModel;
-
-    private \Mautic\CoreBundle\Helper\IpLookupHelper $ipLookupHelper;
-
-    private \Mautic\LeadBundle\Helper\LeadChangeEventDispatcher $leadEventDispatcher;
-
-    private \Mautic\LeadBundle\Twig\Helper\DncReasonHelper $dncReasonHelper;
-
-    private \Doctrine\ORM\EntityManager $entityManager;
-
-    private \Symfony\Contracts\Translation\TranslatorInterface $translator;
-
     /**
      * @var RouterInterface
      */
     private $router;
 
-    /**
-     * Whether or not we're running in a test environment.
-     *
-     * @var bool
-     */
-    private $isTest;
-
     /**
      * @param ModelFactory<object> $modelFactory
+     * @param bool                 $isTest
      */
     public function __construct(
-        IpLookupHelper $ipLookupHelper,
-        AuditLogModel $auditLogModel,
-        LeadChangeEventDispatcher $eventDispatcher,
-        DncReasonHelper $dncReasonHelper,
-        EntityManager $entityManager,
-        TranslatorInterface $translator,
+        private IpLookupHelper $ipLookupHelper,
+        private AuditLogModel $auditLogModel,
+        private LeadChangeEventDispatcher $leadEventDispatcher,
+        private DncReasonHelper $dncReasonHelper,
+        private EntityManager $entityManager,
+        private TranslatorInterface $translator,
         RouterInterface $router,
```

Comment:
@TomasVotruba do you know why Rector omitted this?

Reply:
Not sure, I'd suggest to skip it now. The last PR will add this rule in CI and will enable for all the files. Will check then